christulin/tic-tac-toe
# Feature: Node to DynamoDB Sample

# General Purpose of Pull Request:
Creating a sample of how we can use Node.js w/ Express to call AWS's DynamoDB

# Proposed Changes
This is a stub and PoC. We can work together in understanding it and how we can remove/improve parts of this. 

# How to test?
You'll need to npm install and ensure Node is installed on your computer. 
Once you do that, cd into `node-server` and run `node index.js` - your server should start. 
Next, download PostMan - you'll need it to make calls. 

# Additional Info
https://www.postman.com/
https://nodejs.org/en/
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html
Ref for what I used to understand it: https://gourabp17.github.io/codespace/#/aws/su4dz-qgr1y
https://us-west-2.console.aws.amazon.com/dynamodb/home?region=us-west-2#tables:selected=tic-tac-toe;tab=items
This is where the table lives, if you dont have access hit me up
